client/src/redux/index.js

```diff
@@ -260,7 +263,8 @@ export const reducer = handleActions(
     }),
     [types.openDonationModal]: state => ({
       ...state,
-      showDonationModal: true
+      showDonationModal: true,
+      donationRequested: true
```

Comment:
it would be optimal to have a separate action that switches the donationRequested: true.